requirements.txt

```diff
@@ -3,6 +3,7 @@ lxml           # parser used for ExoMol website
 hjson          # Json with comments (for default_radis.json)
 publib 		   # Plotting styles for Matplotlib. Version, update to 0.4.0 needed for matplotlib==3.8. However, only 0.3.2 is compatible with python==3.8 (see #647)
 hitran-api     # HAPI, used to access TIPS partition functions
+PyQt5
```

Comment:
I agree with your comment in the PR, it could be optional. Maybe you could try the import and if the package is missing, raise an error "Please install PyQt5 with pip install PyQt5"